extensions/optaplanner/runtime/pom.xml

```diff
@@ -0,0 +1,113 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.quarkus</groupId>
+        <artifactId>quarkus-optaplanner-parent</artifactId>
+        <version>999-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>quarkus-optaplanner</artifactId>
+    <name>Quarkus - OptaPlanner - Runtime</name>
+    <description>Constraint solving AI for employee rostering, vehicle routing, maintenance scheduling, school timetabling, etc</description>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-core</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-arc</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-jackson</artifactId>
+            <optional>true</optional>
```

Comment:
Hm, have you tested to make sure this behaves the way you expect it to? And are you sure that `OptaPlannerObjectMapperCustomizer` is never loaded it the user doesn't add `quarkus-jackson`.

@gsmet WDYT about this pattern? I know it's not the way we have been doing things so far, but I do remember @ge0ffrey had a UX issue in mind when doing this, but I don't remember what it was :)

Reply:
I'll check if that customizer is loaded or not. It shouldn't be, but I indeed need to verify.

I am doing it this way, because:
1) Having an extra extension quarkus-optaplanner-jackson-extension (and jaxb, jpa, etc), means users must explicitly add it. That's another hoop to jump through. If they forget to do that, the json marshalling of Score etc defaults to reflection way, which is hard to detect for new users.
2) optaplanner-spring-boot-starter does it this way, it's automagic configuration. No hoop to jump through.

Reply:
Fixed by extracting new quarkus-optaplanner-jackson module